mule-migration-tool-e2e-tests/src/test/resources/e2e/oauth2/tweetbook-oauth2-provider/output/src/main/resources/migration/attributes2inboundProperties.dwl

```diff
@@ -0,0 +1,27 @@
+%dw 2.0
+output application/java
```

Comment:
I think we are not comparing these files in other tests, aren't we? Do we want to add them to all the tests? or remove from this one to keep consistency?